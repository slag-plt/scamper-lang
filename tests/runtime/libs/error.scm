(error "This is an example runtime error")