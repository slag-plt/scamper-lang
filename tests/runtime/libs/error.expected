:1:0: Runtime error:
    A runtime error was encounter: This is an example runtime error
    In program: (error "This is an example runtime error")
